app/controllers/stopwatches_controller.rb

```diff
@@ -2,29 +2,48 @@ class StopwatchesController < ApplicationController
   
   def index
     @stopwatches = current_user.stopwatches
+    respond_to do |format|
+      format.html
+      format.json { render json: @stopwatches}
+    end
   end
 
-  def new
-    @stopwatch = Stopwatch.new
+  def show
+    @stopwatch = Stopwatch.find(params[:id])
   end
 
   def create
     @stopwatch = current_user.stopwatches.create(stopwatch_params)
     if @stopwatch
```

Comment:
same scenario here. You're no verifying the changes were applying correctly the best would be use `@stopwatch.save`, the problem is repeated in your other `update` and `create` actions